### PR TITLE
Mobile skeleton

### PR DESCRIPTION
**Pull request by Bateyjosue**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>mobile skeleton</title>
+  <link rel="stylesheet" href="/style/style.css">
+  <!-- Bootrap Icon -->
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.3/font/bootstrap-icons.css">
+</head>
+<body>
+  <header>
+    <div class="logo">
+      <p>Welcome</p>
```
Comment (vechicin) [inline index.html]: This should be wrapped in an `<a>` tag, please fix it.

Screen Shot 2022-06-30 at 7 52 15

File: index.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    <nav>
+      <a href="">Portfolio</a>
+      <a href="">About</a>
+      <a href="">Contact</a>
+    </nav>
+    <div class="menu">
+      <div></div>
+      <div></div>
+      <div></div>
+    </div>
```
Comment (vechicin) [inline index.html]: This content should be wrapped in a `<nav>` tag.

Screen Shot 2022-06-30 at 7 53 20

File: style/style.css [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+}
+
+#headline {
+  background-image: url("/images/bg1.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-attachment: scroll;
+  height: inherit;
+  padding: 0 20px;
+}
```
Comment (vechicin) [inline style/style.css]: The background image doesn't have the right size. Please take a closer look at the template design and make sure the background image matches the size of the Figma design.

Screen Shot 2022-06-30 at 7 57 59

File: index.html [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+  </header>
+  <main>
+    <section id="headline">
+      <div class="fonts intro">
+        <h2>
+          Hey There. I'm Josh
+        </h2>
```
Comment (vechicin) [inline index.html]: This content should be wrapped in a `<h1>` tag, please fix it.

Screen Shot 2022-06-30 at 7 59 30

File: index.html [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+          I'm a Software Engineer
+        </h2>
+      </div>
+      <div class="more">
+        I can help you build a product, feature or website Look through some of my work and experience!
+        If you like what you see and have a project you need coded, don't hesitate to contect me or 
+        <a href="https://ko-fi.com/https://ko-fi.com/joshbatey" class="coffee"> 
+          <img src="https://cdn.ko-fi.com/cdn/kofi3.png?v=3" height="30" width="110" alt="https://ko-fi.com/joshbatey" />
+        </a>
+      </div>
```
Comment (vechicin) [inline index.html]: This content should be wrapped in a `<p>` tag, please fix it.

Screen Shot 2022-06-30 at 8 01 15

File: index.html [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+        <h2>
+          I'm a Software Engineer
+        </h2>
+      </div>
+      <div class="more">
+        I can help you build a product, feature or website Look through some of my work and experience!
+        If you like what you see and have a project you need coded, don't hesitate to contect me or 
+        <a href="https://ko-fi.com/https://ko-fi.com/joshbatey" class="coffee"> 
+          <img src="https://cdn.ko-fi.com/cdn/kofi3.png?v=3" height="30" width="110" alt="https://ko-fi.com/joshbatey" />
+        </a>
```
Comment (vechicin) [inline index.html]: This content is not present in the template design, please remove it.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+        <a href="#">
+          <i class="bi bi-instagram"></i>
+        </a>
+        <a href="#">
+          <i class="bi bi-facebook"></i>
+        </a>
+        <a href="#">
+          <i class="bi bi-github"></i>
+        </a>
+      </div>
```
Comment (vechicin) [inline index.html]: This content should be wrapped in a `<ul>` with each icon being wrapped in a `<li>`, please fix it.

Screen Shot 2022-06-30 at 8 03 56

Additionally, your icons don't match those of the template design, please make sure you use the same icons.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+    </section>
```
Comment (vechicin) [inline index.html:72]: You're missing the arrow down button at the bottom of the section, please add it.

Screen Shot 2022-06-30 at 8 05 00

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @Bateyjosue ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️

- [ ] Please provide a PR description
- [ ] Be careful with the PR you submit. You submitted a PR to your `Hello Microverse` project, your request could get invalidated as a result

_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Bateyjosue) [reply index.html]: Actually, the design has on the Welcome test

File: style/style.css (hunk 3 above)
Comment (Bateyjosue) [reply style/style.css]: Sorry I don't understand this as long as I Used the background-size: cover; means it should cover the entire width

File: index.html [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>mobile skeleton</title>
+  <link rel="stylesheet" href="/style/style.css">
+  <!-- Bootrap Icon -->
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.3/font/bootstrap-icons.css">
+</head>
+<body>
+  <header>
+    <div class="logo">
+      <a>Welcome</a>
```
Comment (cwaku) [inline index.html]: [OPTIONAL]
- [ ] I suggest you add an `href` to your link here. Generally, it's a good practice that the logo redirects to the homepage when clicked.

File: style/style.css [hunk 10]
```diff
@@ -0,0 +1,118 @@
...
+
+#headline {
+  background-image: url("/images/bg1.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-attachment: scroll;
+  height: inherit;
+  padding: 0 20px;
+  width: 100%;
+}
```
Comment (cwaku) [inline style/style.css]: - [ ] You've done a good job so far, but as required by the previous reviewer kindly get your background image looking closer to the figma design. The intention is to get it as close as possible. Remember the UI is already designed, it's your job to follow it. That's how it is in the real world. A UI designer is hired to do the design that has good UX etc. and the Developer is hired to follow the design (color theme, fonts etc.) as closely as possible.

##### Figma Desgin
image

##### Your design
image

**Review by cwaku — CHANGES_REQUESTED** [review]
Hi @Bateyjosue 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Highlights
- [x] Good commit history 👍🏾 
- [x] All linters passed 💯 
- [x] Meaning PR title 👍🏾 
- [x] Correct use of Github flow 🥇 


## Required Changes ♻️
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips 🌟  that can improve your code quality 💯 and help you get a better job 💼.

I will guide 📔 you through fixing 🛠️ these bugs, follow my comments down below and make sure you don't miss any of them, in order to get your project approved ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

[OPTIONAL] 
- [ ] Kindly add a description to your PR.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style/style.css (hunk 10 above)
Comment (Bateyjosue) [reply style/style.css]: Hi, @cwaku Do you think it's a good practice to copy-paster the code from Figma?

Comment (cwaku) [reply style/style.css]: @Bateyjosue Yes that's totally fine. If your HTML is structured exactly as how the Figma template is, it should work fine

Comment (Bateyjosue) [reply style/style.css]: @cwaku Can you please tell me the way you did to get the result in a mobile display?

Comment (cwaku) [reply style/style.css]: @Bateyjosue I personally didn't choose this template for my portfolio project.

**Pull request merged**